src/reconcile/rec_write.c

```diff
@@ -2004,16 +2004,24 @@ __rec_split_write_reuse(
     if (mod->rec_result != WT_PM_REC_MULTIBLOCK || mod->mod_multi_entries < r->multi_next)
         return (false);
 
+    /*
+     * It is possible that the generated checksum values can be the same from two different images.
+     * Use the time aggregate also along with size and checksum in the comparison to identify
+     * whether the new image is the same as the previously written block to reuse.
```

Comment:
Could you please update the comment here to acknowledge that the check could still give a false positive, even taking timestamps into account?

Reply:
Updated the comment as per your suggestion.